walsamlee/politico
# #163416647 Create-party Endpoint

#### What does this PR do?
Add `create party` API endpoints
#### Description of Task to be completed?
Have it passing unit-testing
#### How should this be manually tested?
- Clone the project, https://github.com/walsamlee/politico
- Check out into the proper branch, `ft-v1-create-party-163416647`
- Using postman, post with endpoint URL `/api/v1/parties` 
#### What are the relevant pivotal tracker stories?
#163416647
